packages/backend-defaults/src/entrypoints/urlReader/lib/GoogleGcsUrlReader.ts

```diff
@@ -56,24 +56,24 @@ const parseURL = (
  *
  * @public
  */
-export class GoogleGcsUrlReader implements UrlReader {
+export class GoogleGcsUrlReader implements UrlReaderService {
   static factory: ReaderFactory = ({ config, logger }) => {
     if (!config.has('integrations.googleGcs')) {
       return [];
     }
     const gcsConfig = readGoogleGcsIntegrationConfig(
       config.getConfig('integrations.googleGcs'),
     );
-    let storage: Storage;
+    let storage: GoogleCloud.Storage;
     if (!gcsConfig.clientEmail || !gcsConfig.privateKey) {
       logger.info(
         'googleGcs credentials not found in config. Using default credentials provider.',
       );
-      storage = new Storage({
+      storage = new GoogleCloud.Storage({
         userAgent: `backstage/backend-common.GoogleGcsUrlReader/${packageinfo.version}`,
```

Comment:
Should prolly update this while at it, to `backend-defaults`, same below

Reply:
I actually thought, what if they are used somewhere :) But yeah OK I changed them now